src/generic/problem.cc

```diff
@@ -11431,11 +11434,20 @@ namespace oomph
                        << std::endl;
           }
           oomph_info
-            << "Note: This behaviour can be adjusted by changing the protected "
+            << "Note: This behaviour can be adjusted by changing the protected"
             << "boolean" << std::endl
+            << "    Problem::Keep_temporal_error_below_tolerance" << std::endl
             << std::endl
-            << "    Problem::Keep_temporal_error_below_tolerance" << std::endl;
+            << "Also, if you are noticing that many of your timesteps result"
+            << "in error > tolerance, try reducing the target error with"
```

Comment:
...should this be '...try reducing the "target error safety factor from its default value of 1.0. This will reduce the next timestep below the default estimate which aims to make the error close to the target error. This default strategy makes the timestep as large as possible while respecting the requested tolerance, but risks that the next timestep is rejected if the error estimate turns out to be too conservative. Setting the safety factor to too small a value will make the timesteps unnecessarily small; choosing a value that is too large may lead to an unnecessarily large number of rejections, forcing a recomputation of the solution at a reduced  timestep. Neither is optimal; a problem dependent compromise is needed; see ***ref**** for details.'

Reply:
I'll add most of what you've said but, importantly, I think the part about
 "...the default estimate which aims to make the error close to the **target** error." 
should be
 "...the default estimate which aims to make the error close to the **tolerance** error."

The idea of the safety factor is that it is changing the target error to be something less than tolerance (provided it is <1.0) whereas the current strategy is to try and produce an error that is equal to our tolerance (which results in **A LOT** of rejected steps *something I want to talk about in oomph-lib* (again)).

However, I agree it is important to clarify the limiting behaviour of too small vs too big in this messge.